### DISCUSSION TITLE
Access to include or copy members with a click like ardí.

### DISCUSSION THREAD

**Discussion by DadVaderDoCode** [codefori/vscode-ibmi#1864] [Q&A]
Hello guys,
First of all code for i rules and it's fantastic.
I just want to know if there's a way to access with a click the include or copys of members declared in a program.

Best Regards,

**Comment by worksofliam** [comment]
You should be able to:

- right click on the copy and go to definition
- control+click with your cursor

**Comment by DadVaderDoCode** [comment]
Great!! Thanks!! 🙌🏻

**Reply by DadVaderDoCode** [reply]
![image](https://github.com/codefori/vscode-ibmi/assets/50391179/949a45c1-6d48-47bc-8ba9-aa39f3856cc5)

I press the right click but nothing happens, i can't see the program in the /include statements.
